posts/setting-up-a-shared-library-and-seed-job-in-jenkins-2.md

```diff
@@ -0,0 +1,189 @@
+
+## Introduction
+In this second part of the series, we will be setting up a [Jenkins Shared library](https://jenkins.io/doc/book/pipeline/shared-libraries/) to execute our Jenkins jobs. As the complexity and number of jobs you maintain grow, the use of Shared Libraries provides the ability to organize and version control your CI and CD processes. 
+
+In order to set up a practical application with our `seedJob`, we will modify `seed.groovy` to build a Pipeline job and a Multibranch Pipeline job. By the end of this series, you will have a foundation set up to onboard projects consistently as well as handle job specific stages during the running of your Jenkins jobs. 
+
+### Source Code
+The source code is available below
+  * Jenkins Shared Library ([`microservice-pipelines`](https://github.com/kcrane3576/microservice-pipelines))
+  * [`poc-micro`](https://github.com/kcrane3576/poc-micro) [JHipster](https://www.jhipster.tech/) microservice
+
+## Prerequisites
+1. Jenkins running in Docker a docker container built from `kcrane121/maven-jenkins:blog`.
+2. `seedJob` set up from Part 1.
+
+## Part 2 Goals
+1. Configure Jenkins to use [`microservice-pipelines`](https://github.com/kcrane3576/microservice-pipelines)
+as our Shared Library for executing jobs. 
+2. Configure `seed.groovy` to create a Pipeline Job and Multibranch Pipeline Job per service.
+3. Create `jenkinsJob.groovy` to be used by our services as the entry point into our Shared Library.
+4. Set up a [`Jenkinsfile`](https://jenkins.io/doc/book/pipeline/jenkinsfile/) in `poc-micro` to use `microservice-pipelines` Shared Library.
+
+## Goal 1
+### Configure default Shared Library setup for Jenkins
+Since we will be using a Shared library, Jenkins needs to be set up to use our Shared Library.
```

Comment:
Maybe you can explain a little more what is a shared library and why you need these steps before.

Reply:
I added a section under Part 2 goals explaining that Shared Libraries become useful when the amount of jobs gets very large and the requirements continuously change